tensorflow/contrib/tensorrt/convert/convert_nodes.h

```diff
@@ -36,11 +38,13 @@ static const char* kInputPHName = "InputPH_";
 static const char* kOutputPHName = "OutputPH_";
 namespace convert {
 
+// TODO(aaroey): use an enum instead.
 const int FP32MODE = 0;
 const int FP16MODE = 1;
 const int INT8MODE = 2;
-struct EngineConnections {
-  EngineConnections(const string& outside, int out_id, int out_port,
+
+struct EngineConnection {
```

Comment:
@aaroey  This structure represent connections of an engine, I believe it should be plural